src/cmd/package.go

```diff
@@ -175,10 +174,10 @@ func choosePackage(args []string) string {
 	return path
 }
 
-func cachePathClean(cachePath string) bool {
-	var isCleanPath = regexp.MustCompile(`^[a-zA-Z0-9\_\-\/\.\~]+$`).MatchString
-	if !isCleanPath(cachePath) {
-		message.Warnf("Invalid characters in Zarf cache path, defaulting to ~/%s", config.ZarfDefaultImageCachePath)
+func isCachePathClean(cachePath string) bool {
+	var isCleanPathRegex = regexp.MustCompile(`^[a-zA-Z0-9\_\-\/\.\~]+$`).MatchString
```

Comment:
No need to change, this pattern sort of bothered me on one level because it's not something I've seen very much in go, reminds me of Javascript prototype reuse--but I do agree it is actually quite readable.  Just a comment that it _feels odd_, but is overall likely more readable so 🤷.

Reply:
slightly refactored. I agree it is odd, we could include the logic in the actual command definition too without this separate function.

Reply:
Removed that separate function